### PR TITLE
[185302168] A script attaching peering security group for RDS

### PR DESCRIPTION
**Pull request by dark5un**
What
----

We introduced a ruby script and make action which allows the creation of a security group against a predefined peering connection. This security group will in turn be attached to a specified RDS database to allow connectivity through that peer.

How to review
-------------

Follow the steps on the story.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: scripts/enable_vpc_peer_db_access.rb [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    abort with_usage
+  end
+  peering_filename = "terraform/#{vpc_name}.vpc_peering.json"
+  unless File.file?(peering_filename)
+    puts "Error: #{peering_filename} not found in path. Exiting"
+    abort with_usage
+  end
+  peering_file_contents = File.read(peering_filename)
+  peering_data = JSON.parse(peering_file_contents)
+  peering_entry = peering_data.find { |x| x["peer_name"] == peer_name }
```
Comment (AP-Hunt) [inline scripts/enable_vpc_peer_db_access.rb:239]: What happens if the peering name isn't found?

**Review by AP-Hunt — CHANGES_REQUESTED** [review]

**Comment by AP-Hunt** [comment]
@dark5un Got a couple comments need addressing here

**Comment by dark5un** [comment]
Review comments have been addressed

**Review by AP-Hunt — APPROVED** [review]
Looks good. Can you squash down the review fixes, please?

**Comment by dark5un** [comment]
A more cleaning up and a new feature:
Providing env var DRY will force the script to run without any alterations to the database.

**Pull request merged**